### ISSUE TITLE
Missing __init__ in namespace packages

### ISSUE THREAD

**Issue by lmdsp** [kbytesys/django-recaptcha2#11]
Hi,

thanks for this great package !

I have found a few minor issues when running with `python -Wall`, namely:

```python
\Python36\lib\importlib\_bootstrap_external.py:426: ImportWarning: Not importing directory env\lib\site-packages\snowpenguin: missing __init__
  _warnings.warn(msg.format(portions[0]), ImportWarning)

\Python36\lib\importlib\_bootstrap_external.py:426: ImportWarning: Not importing directory env\lib\site-packages\snowpenguin\django: missing __init__
  _warnings.warn(msg.format(portions[0]), ImportWarning)

# Repeated about 20 times ...
```
I'm not sure how severe this is, but at least it pollutes the console output, so probably this would be worth fixing ?

Cheers,
Lorcan

**Comment by kbytesys** [comment]
Hi,
every warning is worth to be fixed. This is related to namespace packages and I've just figure out what's wrong in the setuptools configuration.

Thank you for reporting!

**Comment by lmdsp** [comment]
Hi @kbytesys, thanks a lot, great news !

**Comment by kbytesys** [comment]
I think that the issue is solved now, please use the 1.0.3 release and reopen the bug if something is wrong.

Thanks

**Comment by lmdsp** [comment]
I confirm the warnings are all gone now, thanks very much!
